src/clp_ffi_py/Py_utils.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef CLP_FFI_PY_PY_UTILS_HPP
+#define CLP_FFI_PY_PY_UTILS_HPP
+
+#include <clp_ffi_py/Python.hpp>  // Must always be included before any other header files
+
+#include <clp/components/core/src/ffi/encoding_methods.hpp>
+
+namespace clp_ffi_py {
+/**
+ * Initializes CPython interface to Python level utility functions implemented
+ * in submodule `clp_ffi_py.utils`.
+ * @return true on success.
+ * @return false on failure with the relevant Python exception and error set.
+ */
+auto py_utils_init() -> bool;
```

Comment:
If you're prefixing every method name, then this feels like you should have a `py_utils` namespace instead. Right, @davidlion?

Reply:
The only reason we have `py_` here is to follow the convention of C Python, where the python level API should start with `py_`